pom.xml

```diff
@@ -95,6 +91,15 @@
                 <version>${slf4j.version}</version>
             </dependency>
 
+            <!-- Framework support (provided dependencies -->
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
+                <scope>provided</scope>
```

Comment:
I was hoping that not calling the constructor with the TypeToken would be enough to keep the runtime happy, but it's not. I don't think I can make this work as I intended. That's oke though.

I removed the TypeToken altogether.